src/components/Input/BoxedInput.ts

```diff
@@ -71,6 +76,11 @@ const BoxedInput: FC<InternalInputComponentProps> = styled(BaseInput)`
             ${p => activeBoxedPosition(p.size)};
             color: ${p => (p.inverted ? Colors.WHITE : Colors.ACTION_BLUE_900)};
             background: ${p => (p.inverted ? Colors.AUTHENTIC_BLUE_900 : Colors.WHITE)};
+            background: ${p =>
+                `linear-gradient(0deg, ${p.inverted ? Colors.AUTHENTIC_BLUE_900 : Colors.WHITE} calc(50% + ${
+                    // @ts-ignore
+                    p.size === 'small' ? '0.0825rem' : '0.0625rem'
+                }), transparent 50%)`};
```

Comment:
Nit: Maybe it's worth considering to avoid the repetition with the above style as it's exactly the same unless we prefer to have them separate

Reply:
The thing is that I feel I am going to touch in a much bigger refactor. I see that previously the pattern for the background is the same _(duplication)_ and I see that we have some functions to encapsulate some styles. Perhaps, instead of reactively doing something here we may want to step back and think about a better way.